LaTeX/Parts/Chapter4.tex

```diff
@@ -187,54 +191,58 @@ \section{Stages}
 </scenario>
 \end{lstlisting}
 A Scenario is divided into one or more Stages.
-These Stages are executed in sequential order as they are specified in the XML document.
+These Stages are executed sequentially in the order in which they are specified in the XML document.
```

Comment:
```suggestion
These Stages are executed in the order in which they are specified in the XML document.
```
The sequentially just makes it less readable in my opinion.
The reader can figure that out implicitly.